src/Persistence.LinnApps/Repositories/PartRepository.cs

```diff
@@ -37,6 +37,15 @@ public Part FindById(int key)
                 .Include(p => p.MechPartSource)
                 .ThenInclude(m => m.MechPartManufacturerAlts)
                 .ToList().FirstOrDefault();
+            
+            if (result != null)
+            {
+                result.SalesPart =
```

Comment:
must admit i'm not sure this should be in the part repo. Not sure the extra call is warranted for the vast majority of calls that we'll do to parts. On the other hand I get why you've done it.

Reply:
I think you're right. This GetById is taking long enough to return as it is. I will move this step to the domain service update method